dev-requirements.in

```diff
@@ -2,10 +2,8 @@ bandit
 black
 coverage
 django>=3,<4
-flake8==3.8.4
 isort
 model-bakery
 pre-commit
 pylint-django
 prospector[with_vulture]
-safety
```

Comment:
safety wasn't being used, I couldn't find any references to it in the project. It also has version conflicts with other libs so I preferred to just remove it.

Reply:
Wasn't used in CI?

Reply:
No, there isn't any reference in the whole code base